symbulate/random_variables.py

```diff
@@ -11,9 +11,22 @@ def __init__(self, probSpace, fun=lambda x: x):
         self.fun = fun
 
     def draw(self):
+        """A function, that takes no arguments, explaining how to draw one 
```

Comment:
As above, the user wants to know what will happen when they call .draw() on the RV. What arguments does this function take and what does it return?